buildSrc/src/main/kotlin/io/spine/gradle/internal/Scripts.kt

```diff
@@ -0,0 +1,62 @@
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.gradle.internal
+
+import org.gradle.api.Project
+
+@Suppress("unused")
+object Scripts {
+    private const val commonPath = "/config/gradle/"
+
+    fun testArtifacts(p: Project)          = p.script("test-artifacts.gradle")
+    fun testOutput(p: Project)             = p.script("test-output.gradle")
+    fun slowTests(p: Project)              = p.script("slow-tests.gradle")
+    fun javadocOptions(p: Project)         = p.script("javadoc-options.gradle")
+    fun filterInternalJavadocs(p: Project) = p.script("filter-internal-javadoc.gradle")
+    fun jacoco(p: Project)                 = p.script("jacoco.gradle")
+    fun publish(p: Project)                = p.script("publish.gradle")
+    fun publishProto(p: Project)           = p.script("publish-proto.gradle")
+    fun javacArgs(p: Project)              = p.script("javac-args.gradle")
+    fun jsBuildTasks(p: Project)           = p.script("js/build-tasks.gradle")
+    fun jsConfigureProto(p: Project)       = p.script("js/configure-proto.gradle")
+    fun npmPublishTasks(p: Project)        = p.script("js/npm-publish-tasks.gradle")
+    fun npmCli(p: Project)                 = p.script("js/npm-cli.gradle")
+    fun updatePackageVersion(p: Project)   = p.script("js/update-package-version.gradle")
+    fun dartBuildTasks(p: Project)         = p.script("dart/build-tasks.gradle")
+    fun pubPublishTasks(p: Project)        = p.script("dart/pub-publish-tasks.gradle")
+    fun pmd(p: Project)                    = p.script("pmd.gradle")
+    fun checkstyle(p: Project)             = p.script("checkstyle.gradle")
+    fun runBuild(p: Project)               = p.script("run-build.gradle")
+    fun modelCompiler(p: Project)          = p.script("model-compiler.gradle")
+    fun licenseReportCommon(p: Project)    = p.script("license-report-common.gradle")
+    fun projectLicenseReport(p: Project)   = p.script("license-report-project.gradle")
+    fun repoLicenseReport(p: Project)      = p.script("license-report-repo.gradle")
+    fun generatePom(p: Project)            = p.script("generate-pom.gradle")
+    fun updateGitHubPages(p: Project)      = p.script("update-gh-pages.gradle")
+
+    private fun Project.script(name: String) = "${rootDir}${commonPath}${name}"
```

Comment:
As per the errorprone, extracting format strings is considered a bad practice. For me it also takes some extra efforts to go up in the file and read the `commonPath`.

Reply:
This is not introduced by this PR. This code was copied from `deps.kt`. If you'd like to improve on this, I'd be happy to review a PR from you ;)